include/ck/utility/math.hpp

```diff
@@ -152,21 +152,18 @@ __host__ __device__ constexpr T clamp(const T& x, const T& lowerbound, const T&
 
 // disallow implicit type casting
 template <typename T>
-__device__ T exp(T x);
```

Comment:
The original implementation intentionally disallow implicit type casting.

Reply:
Done.